lib/db.js

```diff
@@ -357,11 +371,14 @@ module.exports = (
     log.trace({ op: 'DB.emailRecord', email: email })
     return this.pool.get('/emailRecord/' + hexEncode(email))
       .then(
-        function (data) {
-          data.emailVerified = !! data.emailVerified
-          return data
+        (body) => {
+          return this.accountEmails(body.uid)
+            .then((emails) => {
+              setAccountEmails(body, emails)
```

Comment:
It looks like `setAccountEmails` is always called in this pattern, with `this.accountEmails(body.uid).then(...` before it and `return body` after it. Is it worth pushing those bits inside the method as well to reduce the duplication?